torch/_functorch/aot_autograd.py

```diff
@@ -1049,7 +1049,10 @@ class AOTConfig:
 
 
 def aot_dispatch_base(flat_fn, flat_args: List[Tensor], aot_config: AOTConfig):
-    fw_module = make_fx(flat_fn, aot_config.decompositions)(*flat_args)
+    # flat_args is used by line 951 and line 960
+    # clone flat_args to avoid flat_args shape changed by inplace ops (unsqueeze_)
+    tmp_flat_args = [x.clone() for x in flat_args]
```

Comment:
I believe `clone()` can change the strides etc.  I think there is something in utils to copy a tensor while preserving everything.

cc @ezyang @Chillee on aot_dispatch changes